CMR/python/README.md

```diff
@@ -17,7 +17,26 @@ A wrapper for [github.com/nasa/Common-Metadata-Repository][git_cmr] in python
 
 ## Building and Installing
 
-As of this time the CMR library is not hosted on [pypy.org][pypi]. To install the library from a [wheel][wheel] file it will need to be generated locally using the 'runme.sh' script. On a command line, do the following from the python/CMR directory:
+As of this time the CMR library is not hosted on [pypy.org][pypi]. There are several ways to install the library:
+
+### Stable from github:
+
+***NOTE***: Not yet implemented, as there is not a current stable build yet. When the first version is created, this section will be updated.
+
+### Latest from git hub:
+
+***WARNING***: This will install the latest build and may not be the most stable code!
+
+To install the latest build directly from github, the following command can be.
+used. But be warned, 
+
+    pip3 install 'https://github.com/nasa/eo-metadata-tools/releases/download/latest/eo_metadata_tools_cmr-0.0.1-py3-none-any.whl'{code}
+
+### Install Local Build:
+
+***NOTE***: Preferred method.
+
+To install the library from a local [wheel][wheel] file it will first need to be generated using the 'runme.sh' script. On a command line, run the following from the `python/CMR` directory:
```

Comment:
Correct syntax for a link?

Reply:
yep, I'm using the other syntax to keep the text readable. I think in this case, since the project is open to a larger set of people then something like CMR, using this style is more readable to novice to Markdown, it also allows for hover text to be defined which might be more 508 compliant.